lib/charms/vault_k8s/v0/vault_client.py

```diff
@@ -134,6 +136,11 @@ def get_token_data(self) -> Optional[Dict]:
             return None
         return token_data
 
+    @property
+    def token(self):
```

Comment:
We are missing return typing

Reply:
https://github.com/canonical/vault-k8s-operator/pull/365/commits/e930a2b82f4c32b824a45b1387644aac5b6724ea